src/quacc/recipes/emt/elastic.py

```diff
@@ -15,17 +15,18 @@
 
     from ase.atoms import Atoms
 
-    from quacc.types import OptSchema, RunSchema
+    from quacc.types import ElasticSchema
 
 
 @flow
 def bulk_to_deformations_flow(
     atoms: Atoms,
-    run_static: bool = True,
+    pre_relax: bool = True,
+    run_static: bool = False,
```

Comment:
Ideally, one should know what the keyword arguments mean without having to read the docstring. If these are only for the bulk structure before deformation, we should give them suitable names. For instance, `pre_relax_bulk` and `run_static_bulk` or `pre_relax_undeformed` and `run_static_undeformed` or something like that (with the docstring updated accordingly).